src/pages/academy/components/_search-banner.tsx

```diff
@@ -13,7 +13,7 @@ import { useBrowserResize } from 'components/hooks/use-browser-resize'
 import { slugify, isBrowser } from 'common/utility'
 import { DerivStore } from 'store'
 import device from 'themes/device'
-import AcademyLogo from 'images/svg/blog/academy-logo.svg'
+import AcademyLogo from 'images/svg/blog/academy-text.svg'
```

Comment:
Could you also please remove the original `academy-logo.svg` if it is not in used anymore?

Reply:
@sean Ok noted, will update that. Thank you